src/docs/getting-started/devPackageSource.md

```diff
@@ -0,0 +1,11 @@
+# Add Dev package source
+
+In this article, we are going to add a new package source pointing to the Dev packages.
```

Comment:
Some explanation on what the dev package source is wouldn't hurt. I'm new to Orchard, just reading the getting started guide. What are the dev packages? Why would I use them? What's the difference to the ones on NuGet? Should I use them in production?

Reply:
I don't know what to say about it.

Reply:
Come on :).